style.css

```diff
@@ -3,6 +3,192 @@
   box-sizing: border-box;
 }
 
+/* popupp menu */
+.popup {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  width: 100vh;
+  height: 100vh;
+  overflow-y: scroll;
+}
+
+.section-menu {
+  width: 100%;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  z-index: 99999;
+  background-color: #dededebb;
+  -webkit-backdrop-filter: blur(5px);
+  backdrop-filter: blur(5px);
+}
+
+.popup-btn {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  -ms-flex-pack: distribute;
+  justify-content: space-around;
+}
+
+#close-btn {
+  width: 100%;
+  height: -webkit-fit-content;
+  height: -moz-fit-content;
+  height: fit-content;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  padding-right: 25px;
+}
+
+.popup-header {
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  width: -webkit-fit-content;
+  width: -moz-fit-content;
+  width: fit-content;
+  margin-top: 0;
+}
+
+.card-list-title-popup {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  -webkit-box-flex: 0;
+  -ms-flex: none;
+  flex: none;
+  -webkit-box-ordinal-group: 1;
+  -ms-flex-order: 0;
+  order: 0;
+  -ms-flex-positive: 0;
+  flex-grow: 0;
+  list-style-type: none;
+}
+
+.notice-popup {
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  width: 311px;
+}
+
+.card-popup {
+  height: -webkit-fit-content;
+  height: -moz-fit-content;
+  height: fit-content;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  padding: 16px;
+  width: (100% - 40%);
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  -webkit-box-flex: 0;
+  -ms-flex: none;
+  flex: none;
+  -webkit-box-ordinal-group: 1;
+  -ms-flex-order: 0;
+  order: 0;
+  -ms-flex-positive: 0;
+  flex-grow: 0;
+  margin-top: 60px;
+  margin-left: 16px;
+  margin-right: 16px;
+  margin-bottom: 50px;
+}
+
+.project-button-popup {
+  background-color: #fff;
+  color: #6070ff;
+  font-weight: 500;
+  margin-top: 0;
+  margin-left: 20px;
+  width: 35%;
+  height: 48px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  -ms-flex-pack: distribute;
+  justify-content: space-around;
+}
```

Comment:
- [ ] Great job adding the popup window 👍🏼, but I think it could be better if you stick as much as possible with the Figma template, so please, check the width and height of the buttons, in order to have a similar design with the original. 🦓

Your buttons:

![image](https://user-images.githubusercontent.com/91301423/204098238-2f9b9f95-b71c-467d-a02e-419ec505caaa.png)

Figma buttons:

![image](https://user-images.githubusercontent.com/91301423/204098265-f62a862c-7149-4495-b12a-d178b7441dd9.png)